code/modules/admin/verbs/striketeam.dm

```diff
@@ -1,31 +1,46 @@
-//STRIKE TEAMS
+//Deathsquad
 
-#define COMMANDOS_POSSIBLE 6 //if more Commandos are needed in the future
-GLOBAL_VAR_INIT(sent_strike_team, 0)
+#define MAX_COMMANDOS 14
```

Comment:
Ideally this should not be hardcoded, and instead determined based on the amount of spawners present on the map. However doing so is slightly ugly. Something like

```dm
var/list/ds_spawners = list()
for(var/obj/effect/landmark/spawner/ds/L in GLOB.landmarks_list)
  ds_spawners += L
// and then
var/max_commandos = length(ds_spawners)
```

However, this `define` is an improvement over what used to be here before, and further prettification can be claimed to be out of scope, so feel free to leave it like that.